lua/lvim/bootstrap.lua

```diff
@@ -90,6 +93,22 @@ function M:init(base_dir)
     install_path = self.packer_install_dir,
   }
 
+  -- patch Packer's guess_dir_type to support custom installs using symlinks
+  local guess_dir_type = require("packer.plugin_utils").guess_dir_type
+  require("packer.plugin_utils").guess_dir_type = function(dir)
```

Comment:
This should go into the plugin-loader, since it's the actual wrapper for packer.

Reply:
Probably should go into plugin-loader:init, you're right.